FWCore/Framework/test/test_referencingDeleteEarly_cfg.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TEST")
+
+process.source = cms.Source("EmptySource")
+
+process.maxEvents.input = 3
+
+process.options = cms.untracked.PSet(
+        canDeleteEarly = cms.untracked.vstring("edmtestDeleteEarly_maker__TEST"))
+process.options.holdsReferencesToDeleteEarly = [cms.PSet(product=cms.string("edmtestDeleteEarlyedmRefProd_ref__TEST"), references=cms.vstring(process.options.canDeleteEarly[0]))]
```

Comment:
is this equivalent to
```suggestion
process.options.holdsReferencesToDeleteEarly = [cms.PSet(product=cms.string("edmtestDeleteEarlyedmRefProd_ref__TEST"), references=cms.vstring("edmtestDeleteEarly_maker__TEST"))]
```
?

If they are equivalent, I would find the latter easier to understand and maintain, as soon as there are more than one entry.

Reply:
done